wrappers/src/marshalling.hpp

```diff
@@ -156,6 +167,7 @@ typedef struct realm_value {
         realm_uuid_t uuid;
 
         realm_link_t link;
+        void* collection;
```

Comment:
This can be `CollectionBase*` to be a bit more precise.

Reply:
Done